.pipelines/js-release.yml

```diff
@@ -72,7 +72,8 @@ steps:
       inputs:
         command: publish
         workingDir: source/nodejs/${{ target_package }}
-        publishEndpoint: npmjs.com
+        publishEndpoint: auth_npmjs.com
```

Comment:
can you provide more context on what auth_npmjs.com is?

Reply:
this is a service name that we own. In reality the change made was that the old npmjs.com used a combination of username and password, the new one uses a PAT to make the push